web/siteplan/src/feature/TrackFeature.ts

```diff
@@ -292,38 +293,84 @@ export default class TrackFeature extends LageplanFeature<Track> {
     ])
     const initialTracks = this.getObjectsModel(initial)
     const finalTracks = this.getObjectsModel(final)
-    const isDiffSection = initialTracks.some(initialTrack => {
+    const exsitDiffSection = initialTracks.some(initialTrack => {
```

Comment:
@TruongQuangSB A small typo

```suggestion
    const existDiffSection = initialTracks.some(initialTrack => {
```